lib/address.js

```diff
@@ -255,11 +255,16 @@ Address._transformScript = function(script, network) {
  * @param {Array} publicKeys - a set of public keys to create an address
  * @param {number} threshold - the number of signatures needed to release the funds
  * @param {String|Network} network - either a Network instance, 'livenet', or 'testnet'
+ * @param {boolean=} nestedWitness - if the address uses a nested p2sh witness
  * @return {Address}
  */
-Address.createMultisig = function(publicKeys, threshold, network) {
+Address.createMultisig = function(publicKeys, threshold, network, nestedWitness) {
```

Comment:
`Address.payingTo` is for allowing arbitrary scripts to be passed in. `Address.createMultisig` is for creating multisig p2sh addresses, as bare multisig doesn't have an address, as you mention.